### ISSUE TITLE
ESP8266-2.4.0 core libs are not working

### ISSUE THREAD

**Issue by jgrulich** [Edzelf/Esp-radio#118]
I've updated the ESP8266 core libs to the 2.4.0 and found that it loads the ESP too much and it results in the poor WiFi connection and stream data loss.
I've tested even both rc1 and rc2 with the same result.

**Comment by mischaka** [comment]
I have the same problem

**Comment by Edzelf** [comment]
I have the problem also.  I will try to solve it.

**Comment by Edzelf** [comment]
In the IDE: Select V1.4 Prebuilt as IwIP Variant.  256 kbps stations will work now.

**Comment by jgrulich** [comment]
Not really. I've already tested all the IwIP variants. With all the 2.4.0 versions is the connection very instable and with little bit lower signal level, or in the overloaded WiFi area it fails to connect to some stations at all.
And on the other hand the 2.4.0 lib results in the larger bin file and it loads the CPU significantly more.

**Comment by gitpeut** [comment]
I upgraded to ESP8266 v2.4.1. In 2.4.1 V1.4 Prebuilt is not available. 
After some struggling, I found that compiling with lwip V2.0 Lower Memory worked. 
I also set cpu frequency to 160, but I am not sure how much of a difference that makes.
